File: src/frontend/src/app/models/settings/NoticeSetting.ts
export class NoticeSetting {
    disabledNotice: boolean = false;
    successDuration: number = 2000;
    errorDuration: number = 3000;
    horizontalPosition: 'start' | 'center' | 'end' = 'start';
    verticalPosition: 'top' | 'bottom' = 'bottom';
}

File: src/backend/java/org/dneversky/idea/entity/settings/NoticeSetting.java
package org.dneversky.idea.entity.settings;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Embeddable;

@NoArgsConstructor
@Getter
@Setter
@Embeddable
public class NoticeSetting {

    // may refactor to user's session

    private boolean disabledNotice;
    private int successDuration;
    private int errorDuration;
    private String horizontalPosition;
    private String verticalPosition;
}
